frontend_tests/node_tests/people.js

```diff
@@ -119,8 +121,8 @@ run_test('basics', () => {
     // Reactivating issac
     people.add_in_realm(isaac);
     const active_human_persons = people.get_active_human_persons();
-    assert.equal(active_human_persons.length, 1);
-    assert.deepEqual(active_human_persons, [isaac]);
+    assert.equal(active_human_persons.length, 2);
+    assert.deepEqual(active_human_persons, [me, isaac]);
```

Comment:
This needs a `sort` somewhere to avoid being flaky, right?

Reply:
It's not really flake-prone (no database sequences), but it's not future-proof, so I updated.  Thanks.